lib/Functions.rb

```diff
@@ -0,0 +1,188 @@
+class Functions
+    $debug_on = false
```

Comment:
No usage of **Global variable**

Reply:
Its debug mode just for testing purpose, if im the future i get any bug i can debug more easily